tox.ini

```diff
@@ -12,8 +12,9 @@ requires = virtualenv>=20.16.6
 
 [testenv]
 extras =
-    yara 
+    yara
     mqtt
```

Comment:
We could add these to the dev extra in pyproject.toml as `dissect.target[full,mqtt,yara]`. I think we did similar things for other projects.